voluncheer/map/views.py

```diff
@@ -22,8 +23,23 @@ def map(request):
                 "type": organization_type,
             },
         )
+
+    opportunities = []
+    for opportunity in Opportunity.objects.all():
```

Comment:
We need to filter these opportunities so that we don't include archived opportunities and saved opportunities on the map

Reply:
added those changes!

Reply:
can you tell me where? I don't see the changes

Reply:
It should be in map/views.oh

Reply:
I am still seeing this:
opportunities = []
    for opportunity in Opportunity.objects.all():

Reply:
Let me check once